extensions/amp-story/1.0/amp-story-reaction.css

```diff
@@ -13,3 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+ @import url('https://fonts.googleapis.com/css?family=Poppins:wght@500;700&display=swap');
+
+.i-amphtml-story-reaction {
+  font-family: 'Poppins', sans-serif !important;
+  pointer-events: all;
```

Comment:
Is this new? If yes, let's keep these small changes for followup PRs. It's far too easy to introduce bugs by adding unrelated 1 line changes in a 650+ lines PR. Please file separate issues, or one issue with a list of followup improvements.

Reply:
Added "Bring shared properties to amp-story-reaction.css" into #28530